C#/Botkit/Microsoft.BotKit/BotWorker.cs

```diff
@@ -104,17 +103,39 @@ public async Task<object> Reply(IBotkitMessage message, string response)
         /// </summary>
         /// <param name="id">ID of dialog</param>
         /// <param name="options">Object containing options to be passed into the dialog</param>
-        public async void BeginDialog(string id, object options)
+        public async Task BeginDialogAsync(string id, Dictionary<string, object> options)
```

Comment:
In the Node js code options is a nullable parameter. If we don't want to use default null values in C#, should we leave any kind of indication (in the summary comment, maybe) that the parameter could be null? In case somebody uses this class in the future

Reply:
You're right, I will overload the method so it can also only receive just `id`.